source/docs/contributing/drawio-saving-instructions.rst

```diff
@@ -0,0 +1,10 @@
+Draw.io Saving Instructions
+===========================
+
+This is a explanation of how to save diagrams made in ``draw.io`` (Otherwise known as ``diagrams.net``) as a ``.svg`` file that contains metadata to still be editable in ``draw.io``.
+
+.. image:: images/drawio-example.svg
```

Comment:
Where is there 3 SVGs?

Reply:
Those were just to verify RTD / PDF builds work properly